packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -276,21 +282,21 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
                       : null
                   }
                 >
-                  {isPlaylistUrl(linkValue) ? (
+                  {isCollection ? (
                     <ChatMessagePlaylist
```

Comment:
dang @rickyrombo these components are starting to look mighty similar lol. probs not worth consolidating at this point tho. if it ain't broke don't fix.